ConsoleGame.java

```diff
@@ -0,0 +1,46 @@
+public class ConsoleGame {
+    private final Game game;
+
+    public ConsoleGame(Game game) {
+        this.game = game;
```

Comment:
Also, don't use System.out for Console display, inject an output stream in addition to the input stream.